src/golang/cmd/server/handler/get_artifact_versions.go

```diff
@@ -45,9 +44,9 @@ type artifactVersion struct {
 }
 
 type CheckResult struct {
-	Name     string                   `json:"name"`
-	Status   shared.ExecutionStatus   `json:"status"`
-	Metadata operator_result.Metadata `json:"metadata"`
+	Name     string                 `json:"name"`
+	Status   shared.ExecutionStatus `json:"status"`
+	Metadata shared.ExecutionState  `json:"metadata"`
```

Comment:
I assume changing `Metadata` to `ExecState` would require a migration? Metadata is fine for now then.

Reply:
in either way we would need a migration since the struct is changed. The migration is not tied to this line though, I think check results is not persisted in DB and it's just a response struct.